internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -863,6 +863,16 @@ components:
           type: boolean
         priority:
           type: integer
+    OpenSCAP:
+      type: object
+      required:
+        - datastream
+        - profile_id
+      properties:
```

Comment:
Should this be hard-coded per-distro, and not exposed in the public API?

Reply:
Yeah good point, I'm just confirming with the openscap team that there aren't any cases where there is more than one datastream per distro

Reply:
Cool, going to go ahead and hardcode these now.

Reply:
I'm not that familiar with OSCAP profiles, but does hard-coding means that the caller would not have an option to specify different profiles? Not sure what is the intended workflow, thus asking... :innocent:

Reply:
Yeah, that makes a lot of sense :+1: